book/references.md

```diff
@@ -0,0 +1,9 @@
+# References
+:::{bibliography}
+:::
+
+## External recourses
+
+Parts of this book are taken from other external resources. The following are not edited by the TeachBooks Team:
+
+- [](external/latex-to-markdown-conversion/README.md) {cite:ts}`timon_latex`, not licensed.
```

Comment:
technically if it is not licensed we cannot reuse it. we should ask timon to put a license on it, or get permission to use it.

Reply:
I agree